JavaFXUserInterface/src/main/java/controllers/AssetTypeController.java

```diff
@@ -111,13 +119,110 @@ public void fillAssetTypeTable() {
+                assetTypeName.getStyleClass().add("assetTypeName");
+                Label okLabel = new Label("OK");
+                Label advisoryLabel = new Label("Advisory");
+                Label cautionLabel = new Label("Caution");
+                Label warningLabel = new Label("Warning");
+                Label failedLabel = new Label("Failed");
+
+                HBox okBox = new HBox();
+                okBox.getStyleClass().add("okBox");
+                okBox.getStyleClass().add("ok");
+                okBox.setAlignment(Pos.CENTER);
+                HBox advisoryBox = new HBox();
+                advisoryBox.getStyleClass().add("advisoryBox");
+                advisoryBox.getStyleClass().add("advisory");
+                advisoryBox.setAlignment(Pos.CENTER);
+                HBox cautionBox = new HBox();
+                cautionBox.getStyleClass().add("cautionBox");
+                cautionBox.getStyleClass().add("caution");
+                cautionBox.setAlignment(Pos.CENTER);
+                HBox warningBox = new HBox();
+                warningBox.getStyleClass().add("warningBox");
+                warningBox.getStyleClass().add("warning");
+                warningBox.setAlignment(Pos.CENTER);
+                HBox failedBox = new HBox();
+                failedBox.getStyleClass().add("failedBox");
+                failedBox.getStyleClass().add("failed");
+                failedBox.setAlignment(Pos.CENTER);
+                Text okAssets = new Text(String.valueOf(assetType.getCountOk()));
+                Text advisoryAssets = new Text(String.valueOf(assetType.getCountAdvisory()));
+                Text cautionAssets = new Text(String.valueOf(assetType.getCountCaution()));
+                Text warningAssets = new Text(String.valueOf(assetType.getCountWarning()));
+                Text failedAssets = new Text(String.valueOf(assetType.getCountFailed()));
+
+                assetTypeName.setLayoutX(15.0);
+                assetTypeName.setLayoutY(35.0);
+                okLabel.setLayoutX(15.0);
+                okLabel.setLayoutY(76.0);
+                advisoryLabel.setLayoutX(15.0);
+                advisoryLabel.setLayoutY(119.0);
+                cautionLabel.setLayoutX(15.0);
+                cautionLabel.setLayoutY(162.0);
+                warningLabel.setLayoutX(15.0);
+                warningLabel.setLayoutY(205.0);
+                failedLabel.setLayoutX(15.0);
+                failedLabel.setLayoutY(248.0);
+
+                okBox.setLayoutX(131.0);
+                okBox.setLayoutY(68.0);
+                advisoryBox.setLayoutX(131.0);
+                advisoryBox.setLayoutY(111.0);
+                cautionBox.setLayoutX(131.0);
+                cautionBox.setLayoutY(154.0);
+                warningBox.setLayoutX(131.0);
+                warningBox.setLayoutY(197.0);
+                failedBox.setLayoutX(131.0);
+                failedBox.setLayoutY(240.0);
```

Comment:
There's so much code for handling the generating the layout in the controller... I think it would be better to have a helper class to handle those operations and extract this code there.